infra/feast-operator/internal/controller/featurestore_controller_test.go

```diff
@@ -216,6 +217,74 @@ var _ = Describe("FeatureStore Controller", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(controllerutil.HasControllerReference(svc)).To(BeTrue())
 			Expect(svc.Spec.Ports[0].TargetPort).To(Equal(intstr.FromInt(int(services.FeastServiceConstants[services.OnlineFeastType].TargetHttpPort))))
+
+			// change projectDir to use a git repo
```

Comment:
Does this test cover all the new options? the feast init parameters and github ?

Reply:
this one tests the git options... the next one tests the init ones -
https://github.com/feast-dev/feast/blob/d64f01ecf406f9d429196ae0bd08a84ec90f0c15/infra/feast-operator/internal/controller/featurestore_controller_test.go#L265